frontend/src/pages/RecurringRunDetailsV2.test.tsx

```diff
@@ -98,7 +98,9 @@ describe('RecurringRunDetailsV2', () => {
     };
 
     jest.clearAllMocks();
-    jest.spyOn(features, 'isFeatureEnabled').mockReturnValue(true);
+    jest
+      .spyOn(features, 'isFeatureEnabled')
+      .mockImplementation(featureKey => featureKey === features.FeatureKey.V2_ALPHA);
```

Comment:
Non-blocking: I think we can start removing V2_ALPHA flag from the codebase, but only keeping its definition as comment in `features.ts` to avoid naming conflict in the future. Of course it can be addressed in a different PR.

Reply:
Will work this is separate PR